jetty-home/src/main/resources/modules/logging-slf4j.mod

```diff
@@ -0,0 +1,18 @@
+# DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
+
+[description]
+Configure logging to use slf4j with no impl
+(If you don't select an impl, then NOP will be used by slf4j)
+
+[tags]
+logging
+
+[provides]
+slf4j
```

Comment:
I think this module should just be called `slf4j` not ``logging-sl4j` as it doesn't provide logging, just a common dependency used by all logging.